### PR TITLE
Edits in geom_series.rst 

### PR DESCRIPTION
**Pull request by mtiley**
- code repetition reduced 
- typos fixed

### CODE REVIEW

File: source/rst/geom_series.rst [hunk 1]
```diff
@@ -859,54 +855,52 @@ i.e., the fraction of income that is consumed
...
+    values = [0.3, 0.4]
+
+    for i in values:
+        y = calculate_y(i, b, g_0, T, y_init)
+        ax1.plot(x, y, label=f"i={i}")
+    for g in values:
+        y = calculate_y(i_0, b, g, T, y_init)
+        ax2.plot(x, y, label=f"g={g}")
+
+    plots = [ax1, ax2]
```
Comment (jstac) [inline source/rst/geom_series.rst]: here and in the next line, you could omit the square brackets and still have the same outcome.

**Comment by jstac** [comment]
This is very nice @mtiley.  Please see my minor comment.

File: source/rst/geom_series.rst (hunk 1 above)
Comment (mtiley) [reply source/rst/geom_series.rst]: Thanks -- new commit should have fixed that now.

File: source/rst/geom_series.rst [hunk 2]
```diff
@@ -859,54 +855,52 @@ i.e., the fraction of income that is consumed
...
+    values = [0.3, 0.4]
+
+    for i in values:
+        y = calculate_y(i, b, g_0, T, y_init)
+        ax1.plot(x, y, label=f"i={i}")
+    for g in values:
+        y = calculate_y(i_0, b, g, T, y_init)
+        ax2.plot(x, y, label=f"g={g}")
+
+    plots = ax1, ax2
```
Comment (jstac) [inline source/rst/geom_series.rst]: please change `plots` to `axes`

File: source/rst/geom_series.rst [hunk 3]
```diff
@@ -859,54 +855,52 @@ i.e., the fraction of income that is consumed
...
+    param_labels = "Investment", "Government Spending"
+    for pair in zip(plots, param_labels):
```
Comment (jstac) [inline source/rst/geom_series.rst]: please change `pair` to `ax, label` and then drop the next two lines.  Inside the loop, please make the appropriate name changes.

**Comment by jstac** [comment]
@mtiley, please add the "ready" label when this is ready for review again.

**Pull request merged**